mass_mailing_sending_queue/models/mail_mass_mailing.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models, _
+from openerp.exceptions import Warning as UserError
+
+
+class MailMassMailingSending(models.Model):
+    _inherit = 'mail.mass_mailing'
+
+    state = fields.Selection(selection_add=[
+        ('sending', "Sending"),
+    ])
+    pending = fields.Integer(string="Pending", compute='_compute_pending')
+
+    @api.multi
+    def send_mail(self):
+        # Override 'Send to All' button completly.
+        # NOTE: Please, notice that this breaks inheritance chain
```

Comment:
[Looking at the code, I guess the part that gets stuck is `mail.compose.message.send_mail`](https://github.com/odoo/odoo/blob/8.0/addons/mass_mailing/models/mass_mailing.py#L614).

I imagine you could simply call super with a context like `.with_context(quick_queue=True)` and then overload `mail.compose.message.send_mail` and change behavior when its context contains that flag. Is that possible?